packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -59,3 +59,21 @@ def test_ip_per_container(cluster):
         app_port = app_definition['container']['docker']['portMappings'][0]['containerPort']
         cmd = '/opt/mesosphere/bin/curl -s -f http://{}:{}/ping'.format(service_points[1].ip, app_port)
         ensure_routable(cmd, service_points)()
+
+
+def test_ip_per_container_with_named_vip(cluster):
+    """Test if we are able to connect to a task with ip-per-container mode
+    """
+    # Launch the test_server in ip-per-container mode
+
+    app_definition, test_uuid = cluster.get_test_app_in_docker(ip_per_container=True)
```

Comment:
ip_per_container in this app def exposes the hostPort on the overlay. We need to create a app that has a port only on the overlay.